apps/src/code-studio/pd/application/teacher2021/Section4ProfessionalLearningProgramRequirements.jsx

```diff
@@ -301,22 +313,23 @@ export default class Section4SummerWorkshop extends LabeledFormComponent {
                 . Let us know if your school would be able to pay the fee or if
                 you need to be considered for a scholarship.
               </label>
+              {this.singleCheckboxFor('understandFee')}
               {this.radioButtonsFor('payFee')}
               {showPayFeeNote && (
```

Comment:
@islemaster broken how? just that it's not checked, or am I not seeing something else?

Reply:
![image](https://user-images.githubusercontent.com/1615761/65714428-e9bc9800-e04f-11e9-9b98-b30915b2e319.png)

Right here, we've clicked "No" - I won't be able to pay the fee - but the additional note isn't showing up.  And in an earlier screenshot it looks like we did check "Yes, I plan to teach..."{

Here's the payFee logic:
https://github.com/code-dot-org/code-dot-org/blob/e822ae3ee7d14364ed93cbf6da12c12c499f6199/apps/src/code-studio/pd/application/teacher1920/Section4ProfessionalLearningProgramRequirements.jsx#L210-L220

Haven't manually tested this myself though.

Reply:
Oh, I see! That's just an inconsistency in the screenshots. I just double checked and it is working. Good catch!

Reply:
I could be reading that logic wrong, but my impression is that this is working as expected. The note should show if:
* Answer to "I plan to teach..." is anything _other_ than "Yes"
AND
* Answer to "Will your school be able to pay the fee?" is "No..."

Since the first condition is false and the second condition is true, the warning should not be displayed.